### PR TITLE
[Toolkit Migration] Remaining samples and resolved merged conflicts.

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
Included Samples

App Complete Auth - Nodejs
Tab-graph-toolkit -Nodejs
Bot-release-management -Nodejs
App SSO -Nodejs

### CODE REVIEW

File: samples/app-sso/nodejs/.vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            "label": "Start application",
+            "dependsOn": [
+                "Start frontend",
+                "Start backend"
+            ]
+        },
+        {
+            "label": "Start frontend",
+            "type": "shell",
+            "command": "npm run dev:teamsfx",
```
Comment (formulahendry) [inline samples/app-sso/nodejs/.vscode/tasks.json:90]: In samples/app-sso/nodejs/client/package.json, there is not `dev:teamsfx` in scirpts section. Will it start the frontend?

**Review by formulahendry — APPROVED** [review]

File: samples/app-sso/nodejs/.vscode/tasks.json (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/app-sso/nodejs/.vscode/tasks.json:90]: fixed

File: samples/app-complete-auth/nodejs/README.md [hunk 2]
```diff
@@ -119,6 +133,14 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 ### 2. Setup NGROK
 1) Run ngrok - point to port 3978
 
+    ```bash
+   ngrok http 3978 --host-header="localhost:3978"
+   ```
+   Alternatively, you can also use the `dev tunnels`. Please follow [Create and host a dev tunnel](https://learn.microsoft.com/en-us/azure/developer/dev-tunnels/get-started?tabs=windows) and host the tunnel with anonymous user access command as shown below:
+
+   ```bash
+   devtunnel host -p 3333 --allow-anonymous
```
Comment (ChetanSharma-msft) [inline samples/app-complete-auth/nodejs/README.md:142]: Why dev tunnel using port ? 3333

ngRok you mentioned 3978

File: samples/app-complete-auth/nodejs/index.js [hunk 3]
```diff
@@ -14,20 +14,10 @@ const axios = require('axios')
 // Read botFilePath and botFileSecret from .env file.
 const ENV_FILE = path.join(__dirname, '.env');
 require('dotenv').config({ path: ENV_FILE });
-const PORT = process.env.PORT || 3978;
-const server = express();
```
Comment (ChetanSharma-msft) [inline samples/app-complete-auth/nodejs/index.js:18]: Why did you remove this part?

File: samples/bot-release-management/nodejs/helpers/devOpsHelper.js [hunk 4]
```diff
@@ -28,7 +28,6 @@ class DevOpsHelper {
             AssignedToProfileImage: isAssignedToPresent ? await graphHelper.GetProfilePictureByUserPrincipalNameAsync(DevOpsHelper.GetEmailOrName(workItem.resource.fields[Constant.AssignedTo], false)) : "",
             CreatedByName: DevOpsHelper.GetEmailOrName(workItem.resource.fields[Constant.CreatedByKey], true),
             CreatedByProfileImage: image,
-            StakeholderTeam: DevOpsHelper.ValidateMails(workItem.resource.fields[Constant.StakeHolderTeamKey].split(',')),
```
Comment (ChetanSharma-msft) [inline samples/bot-release-management/nodejs/helpers/devOpsHelper.js:31]: Why did you remove email validation part?

File: samples/tab-graph-toolkit/nodejs/src/index.tsx [hunk 5]
```diff
@@ -1,13 +1,19 @@
...
-  scopes: ['calendars.read', 'user.read', 'openid', 'profile', 'people.read', 'user.readbasic.all']
+import * as MicrosoftTeams from "@microsoft/teams-js";
+import { Providers } from '@microsoft/mgt';
+import { TeamsMsal2Provider } from '@microsoft/mgt-teams-msal2-provider';
+import './index.css';
+
+TeamsMsal2Provider.microsoftTeamsLib = MicrosoftTeams;
+
+Providers.globalProvider = new TeamsMsal2Provider({
+  clientId: '2c865108-9c5e-456c-94c3-bb792a77576b',
```
Comment (ChetanSharma-msft) [inline samples/tab-graph-toolkit/nodejs/src/index.tsx:12]: Why you have hardcoded client id??
How are we going to get and update it, please check and update readme if needed.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments

File: samples/app-complete-auth/nodejs/index.js (hunk 3 above)
Comment (Mohammed-MSFT) [reply samples/app-complete-auth/nodejs/index.js:18]: it was not removed but, just changes the position 
image

File: samples/bot-release-management/nodejs/helpers/devOpsHelper.js (hunk 4 above)
Comment (Mohammed-MSFT) [reply samples/bot-release-management/nodejs/helpers/devOpsHelper.js:31]: In feilds object there was some properties, and the property **StakeHolderTeamKey** is not exist and its throwing error while splitting the value using split(',')

Email is exists in this property **CreatedByKey**.

So that line of code which i have removed is deprecated.

File: samples/tab-graph-toolkit/nodejs/src/index.tsx (hunk 5 above)
Comment (Mohammed-MSFT) [reply samples/tab-graph-toolkit/nodejs/src/index.tsx:12]: Actually I faced error on this line, while testing sample I was running app without updating client id here... because in readme it was not mentioned, by this error I came to know we need to pass client id.

I have updated In Readme as well.

Thanks for this comment.

File: samples/app-complete-auth/nodejs/README.md (hunk 2 above)
Comment (Mohammed-MSFT) [reply samples/app-complete-auth/nodejs/README.md:142]: duplicated, Fixed this comment

File: samples/bot-release-management/nodejs/helpers/devOpsHelper.js [hunk 6]
```diff
@@ -11,7 +11,7 @@ class DevOpsHelper {
     static async MapToReleaseManagementTask (workItem) {
         var graphHelper = new GraphHelper();
         var isAssignedToPresent = workItem.resource.fields.hasOwnProperty(Constant.AssignedTo);
-        var emailList = workItem.resource.fields[Constant.StakeHolderTeamKey].split(',');
+        var emailList = [];
```
Comment (Prithvi-MSFT) [inline samples/bot-release-management/nodejs/helpers/devOpsHelper.js:14]: why empty ?

**Review by Prithvi-MSFT — CHANGES_REQUESTED** [review]

**Review by ChetanSharma-msft — APPROVED** [review]
Looks fine, approving it.

File: samples/bot-release-management/nodejs/helpers/devOpsHelper.js (hunk 6 above)
Comment (Mohammed-MSFT) [reply samples/bot-release-management/nodejs/helpers/devOpsHelper.js:14]: image

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Pawank-MSFT** [comment]
Executed build pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6350574935

**Comment by Mohammed-MSFT** [comment]
**Executed build Pipeline :** https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6414798750
